azure-kusto-data/azure/kusto/data/_token_providers.py

```diff
@@ -30,6 +30,10 @@ class AsyncManagedIdentityCredential:
         def __init__(self):
             raise KustoAioSyntaxError()
 
+    class AsyncAzureCliCredential:
+        def __init__(self):
+            raise KustoAioSyntaxError()

```

Comment:
nit: Add a comment since I needed you to explain this to me when w ereviewed it together

Reply:
Added, WDYT?